docker-compose/camunda-8.6/README.md

```diff
@@ -152,31 +156,27 @@ If you enabled authentication for GRPC requests on Zeebe you need to provide cli
 
 ## Web Modeler Self-Managed
 
-> :information_source: Web Modeler Self-Managed is available to Camunda enterprise customers only.
+> :information_source: Web Modeler is currently restricted to five collaborators per project. Read more about the restriction [here](https://docs.camunda.io/docs/components/modeler/web-modeler/collaboration/).
```

Comment:
I could update this later when I can point to other documentation spots about the license change.

Reply:
Let's add this to a follow-up issue so that we don't forget about adjusting this